DB/src/functions.js

```diff
@@ -25,6 +25,15 @@ module.exports = {
 		  });
 	},
 
+	getReport: function(connection, logger, rep, callback){
+		connection.query(`select * from reports where id = ?`, [rep], function(err, rows, fields){
```

Comment:
Maybe just change parameter "rep" to something easier to understand like "repID" or "reportID"